docs/usage/projects.md

```diff
@@ -56,21 +57,23 @@ The list of members (again a list in `.spec.members[]` using the `rbac.authoriza
 Each project member must have at least one role (currently described in `.spec.members[].role`, additional roles can be added to `.spec.members[].roles[]`). The following roles exist:
 
 * `admin`: This allows to fully manage resources inside the project (e.g., secrets, shoots, configmaps, and similar).
+* `saAdmin`: This allows to fully manage service accounts inside the project and request tokens for them.
```

Comment:
I would prefer to call this `serviceaccountmanager` or something like this (`sa` is not too expressive; avoid camel case; avoid `admin` which already exists).

Reply:
> I would prefer to call this `serviceaccountmanager` or something like this

`serviceaccountmanager` is too long. What do you think about 'sa-manager'? I think people will guess that 'sa-' stands for 'service account'.

Reply:
Why is it too long? I think it's preferable to have self-explaining names if possible. Unless there is a technical length restriction that prevents us from using "serviceaccountmanager" I would use it since you can understand right away what it is about.

Reply:
Ok. Thanks!

Reply:
I also vote for 'serviceaccountmanager'

Reply:
`serviceaccountmanager` sounds good to me! I will make the change